### PR TITLE
Completed Integration of all modules

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
## Description
Completed Integration of all modules
Works end to end

### CODE REVIEW

File: scripts/build/Vstest.Coverage.Settings.targets [hunk 1]
```diff
@@ -52,6 +52,9 @@
         <PackageReference Include="Microsoft.NET.Test.Sdk">
           <Version>$(NETTestSdkPreviousVersion)</Version>
         </PackageReference>
+        <PackageReference Include="Microsoft.NET.Test.Sdk">
```
Comment (smadala) [inline scripts/build/Vstest.Coverage.Settings.targets]: This is dup.

File: src/CodeCoverageToLcovConverter/CodeCoverageDataModel.cs [hunk 2]
```diff
@@ -16,4 +16,4 @@ public CodeCoverageDataModel()
         // List of Ranges
         public List<Range> Ranges { get; set; }
     }
-}
+}
```
Comment (smadala) [inline src/CodeCoverageToLcovConverter/CodeCoverageDataModel.cs]: Line endiing.

File: src/CodeCoverageToLcovConverter/CodeCoverageReader.csproj [hunk 3]
```diff
@@ -1,8 +1,6 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (smadala) [inline src/CodeCoverageToLcovConverter/CodeCoverageReader.csproj]: Good change 👍

File: src/Coverage.Report/CLIReportTask.cs [hunk 4]
```diff
@@ -12,18 +12,27 @@ public CLIReportTask() : this(new Trace())
...
         }
 
         public override bool Execute()
         {
-            this.trace.Log("=================================================" + Environment.NewLine +
-                           "|   Module                     |  Coverage      |" + Environment.NewLine +
-                           "=================================================" + Environment.NewLine +
-                           "| Spekt.Vstest.Coverage.Report |  99.99%        |" + Environment.NewLine +
-                           "=================================================");
+            //this.trace.Log("=================================================" + Environment.NewLine +
```
Comment (smadala) [inline src/Coverage.Report/CLIReportTask.cs]: Remove code.

File: src/Coverage.Report/Spekt.VSTest.Coverage.Report.targets [hunk 5]
```diff
@@ -10,14 +10,12 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
...
-      <!--<Output PropertyName="VSTestLogger" TaskParameter="Value"/>-->
-    <!--</CreateProperty>-->
-  <!--</Target>-->
-
-  <Target Name="CoverageReportPost" AfterTargets="VSTest">
+  <Target Name="CoverageReportPre" BeforeTargets="VSTest">
+    <CreateProperty Value="$(VSTestLogger);CoverageLogger">
+      <Output PropertyName="VSTestLogger" TaskParameter="Value"/>
+    </CreateProperty>
     <Message Text="Parsing logs here..." Importance="High" />
```
Comment (smadala) [inline src/Coverage.Report/Spekt.VSTest.Coverage.Report.targets]: Add appropriate message.

File: scripts/build/Vstest.Coverage.Settings.targets (hunk 1 above)
Comment (ShreyasRmsft) [reply scripts/build/Vstest.Coverage.Settings.targets]: fixed

File: src/CodeCoverageToLcovConverter/CodeCoverageDataModel.cs (hunk 2 above)
Comment (ShreyasRmsft) [reply src/CodeCoverageToLcovConverter/CodeCoverageDataModel.cs]: fixed

File: src/Coverage.Report/CLIReportTask.cs (hunk 4 above)
Comment (ShreyasRmsft) [reply src/Coverage.Report/CLIReportTask.cs]: done

**Pull request merged**